packages/tools/package/test/module/changelog/changelog.utils.test.js

```diff
@@ -22,6 +22,9 @@ describe('changelogUtils', () => {
   fs.readFileSync.mockReturnValue(fixtures.changelogData);
   describe('createOrUpdateChangelog', () => {
     const filePath331 = fixtures.packagesData[0].version.split('-')[0].replace(/\./g, '_');
+    const filePath000 = fixtures.packagesData[1].data.packageInfo.version
```

Comment:
What's the reasoning behind the `000`?

Just curious!

Reply:
the version is 0.0.0-next.